src/components/ProfileHoverCard/index.web.tsx

```diff
@@ -108,33 +125,55 @@ export function ProfileHoverCardInner(props: ProfileHoverCardProps) {
           },
         }
       }
-
-      // We'll make a decision at the end of a grace period timeout.
       if (state.stage === 'might-show') {
-        if (action === 'unhovered') {
+        // We'll make a decision at the end of a grace period timeout.
+        if (action === 'unhovered-target' || action === 'unhovered-card') {
           return hidden()
         }
         if (action === 'hovered-long-enough') {
-          return showing()
+          return showing({
+            reason: state.reason,
+          })
         }
       }
 
       // --- Showing ---
       // The card is beginning to show up and then will remain visible.
-      function showing(): State {
-        return {stage: 'showing'}
+      function showing({
+        reason,
+      }: {
+        reason: 'hovered-target' | 'hovered-card'
+      }): State {
+        return {
+          stage: 'showing',
+          reason,
+          effect() {
+            function onScroll() {
+              dispatch('scrolled-while-showing')
+            }
+            window.addEventListener('scroll', onScroll)
+            return () => window.removeEventListener('scroll', onScroll)
+          },
+        }
       }
-
-      // If the user moves the pointer away, we'll begin to consider hiding it.
       if (state.stage === 'showing') {
-        if (action === 'unhovered') {
-          return mightHide(HIDE_DELAY)
+        // If the user moves the pointer away, we'll begin to consider hiding it.
+        if (action === 'unhovered-target' || action === 'unhovered-card') {
+          return mightHide()
+        }
+        // Scrolling away if the hover is on the target instantly hides without a delay.
+        // If the hover is already on the card, we won't this.
+        if (
+          state.reason === 'hovered-target' &&
+          action === 'scrolled-while-showing'
+        ) {
+          return hiding()
```

Comment:
Really only mentioning this because you brought it up in the PR.

I don't feel super partial to either way of doing this - scrolling would almost always take your cursor off the card anyway so it's going to be hidden - but if you personally felt it would be nicer to disappear on any scrolling I think it would be safe. I think there might be something to letting scroll override anything - it's a different interaction than viewing the card.

Cards are never going to appear off screen and again, almost any amount of scroll will result in the card being hidden given how small they are. Up to you! (Or if I'm overlooking some reason that you'd definitely need to not hide on scroll...maybe I am)

Reply:
I actually started with this, and then I felt uncomfortable that scrolling dismissed a card I was hovering. I then checked what Twitter does, and it special-cases this. So I think maybe it makes sense to special-case.

Reply:
cool, sgtm